### PR TITLE
Add kaspresso app module

### PR DESCRIPTION
**Pull request by RuslanMingaliev**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -66,6 +66,9 @@ subprojects {
             forRepository {
                 google()
             }
+            forRepository {
```
Comment (RuslanMingaliev) [inline build.gradle.kts:69]: `com.google.android.apps.common.testing.accessibility.framework:accessibility-test-framework:2.0` лежит в `mavenCentral`. Если есть другое решение, подскажите. Отдельный блок `exclusiveContent` у меня не завелся почему-то.

File: subprojects/android-test/test-app-kaspresso/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (eugene-krivobokov) [inline subprojects/android-test/test-app-kaspresso/.gitignore]: No need. We have it in the root `.gitgnore` config

File: subprojects/android-test/test-app-kaspresso/src/androidTest/kotlin/com/avito/android/kaspressoui/test/screen/MainScreen.kt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import com.agoda.kakao.text.KButton
+import com.agoda.kakao.text.KTextView
+import com.avito.android.kaspressoui.MainActivity
+import com.avito.android.kaspressoui.R
+import com.kaspersky.kaspresso.screens.KScreen
+import com.kaspersky.kaspresso.testcases.core.testcontext.BaseTestContext
+
+object MainScreen : KScreen<MainScreen>() {
+
+    override val layoutId: Int? = R.layout.activity_main
```
Comment (eugene-krivobokov) [inline subprojects/android-test/test-app-kaspresso/src/androidTest/kotlin/com/avito/android/kaspressoui/test/screen/MainScreen.kt:12]: It's good. We can try an impact analysis on Kaspresso page objects.
@dsvoronin , cc

File: subprojects/android-test/test-app-kaspresso/.gitignore (hunk 2 above)
Comment (RuslanMingaliev) [reply subprojects/android-test/test-app-kaspresso/.gitignore]: Didn't notice, removed the file

**Comment by sergioboi** [comment]
Execution failed for task ':subprojects:android-test:test-app-kaspresso:compileStagingAndroidTestKotlin'.
@RuslanMingaliev let me make it work
But we need to realize how can I access you fork branch

File: build.gradle.kts (hunk 1 above)
Comment (sergioboi) [reply build.gradle.kts:69]: The problem depends on behavior of exclusiveContent. There is no way to have exclusive contents with the same filters

**Review by sergioboi — APPROVED** [review]

File: subprojects/android-test/test-app-kaspresso/src/androidTest/kotlin/com/avito/android/kaspressoui/test/screen/MainScreen.kt (hunk 3 above)
Comment (matzuk) [reply subprojects/android-test/test-app-kaspresso/src/androidTest/kotlin/com/avito/android/kaspressoui/test/screen/MainScreen.kt:12]: @eugene-krivobokov I suggest to create a task and put it in our Avokado backlog. What do you think?

**Pull request merged**